doc/guides/nest2_to_nest3/nest2_to_nest3_detailed_transition_guide.rst

```diff
@@ -13,11 +13,6 @@ NEST 3.0: Detailed transition guide
    :depth: 2
 
 
-**Dependencies:**
-
- * Python 3.X
- * Matplotlib 3.X.X
```

Comment:
Have we written clearly that with NEST 3.0 we will no longer support Python 2? I see that there is a sentence about it in the `nest2_to_nest3_overview` guide, stating `With NEST 3.0, we no longer support Python 2, which reached its end of life on January 1, 2020.`, but I am wondering if we should state this even more clearly? It is a bit hidden under [*Whats changed*](https://nest-simulator.readthedocs.io/en/latest/guides/nest2_to_nest3/nest2_to_nest3_overview.html#what-s-changed) in the guide and users might not spot it. It might be a problem if users are already using NEST and don't look at the install page for dependencies. I agree that it should not be written under *Dependencies* in the transition guide though, but I liked that it was written at the top of the page. I don't know if it fits best under the transition guide or under the overview.

Reply:
Hi @stinebuu, that is a very good point. I believe it makes more sense to state this point within a "Note" in the ``NEST 3.0: What's new?`` overview, since this guide has top-level visibility in the navigation. We could also add a bullet point on top of the detailed transition guide, in case a user doesn't see it in the overview guide. How does that sound?

Reply:
@sarakonradi This sounds good! I don't know if we need it both places, but lets do it so that we are sure.